taichi/backends/opengl/shaders/print.glsl.h

```diff
@@ -0,0 +1,13 @@
+// vim: ft=glsl
+// clang-format off
+#include "taichi/util/macros.h"
+STR(
+void _msg_push_i32(int x) {
+  int i = atomicAdd(_msg_count_, 1);
```

Comment:
There could be a buffer overflow? How about move the CPU side check to here, i.e. write `x` only if `i < MAX_PRINT_ENTRIES`